src/modules/timezones/entities/timezone.entity.ts

```diff
@@ -0,0 +1,10 @@
+import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class Timezone {
```

Comment:
Is this a standalone schema? Could not find it here - https://app.eraser.io/workspace/d1jC73I9ARAXST9QM2do

Reply:
yeah, it's a standalone schema